readme.markdown

```diff
@@ -727,7 +727,7 @@ installing your package (on macOS):
 ```
   Hint: This package requires some external dependencies.
   Hint: To install them you may be able to run:
-  Hint:   sudo brew install openssl
```

Comment:
This is taken straight from Nimble's output, modifying this is really not the right thing to do. Fix the output.

Reply:
i don't understand; here's what I'm seeing:

drange.nimble:
```
import distros
foreignDep "openssl"
```
nimble install
```
nimble install
  Verifying dependencies for drange@0.1.0
 Installing drange@0.1.0
    Prompt: drange@0.1.0 already exists. Overwrite? [y/N]
    Answer: y
   Success: drange installed successfully.
      Hint: This package requires some external dependencies.
      Hint: To install them you may be able to run:
      Hint:   brew install openssl
```
=> it doesn't show `sudo`